### ISSUE TITLE
Emergency compliments failing in CI intermittently

### ISSUE THREAD

**Issue by jaraco** [pmxbot/pmxbot#97]
In [this run](https://github.com/pmxbot/pmxbot/actions/runs/1118131469), observe that `test_emergency_compliment` fails on Python 3.6 but not on 3.9. In a earlier run, it failed on 3.9. It's passing for me locally. Interestingly, the error is 404:

```
self = <Response [404]>

    def raise_for_status(self):
        """Raises :class:`HTTPError`, if one occurred."""
    
        http_error_msg = ''
        if isinstance(self.reason, bytes):
            # We attempt to decode utf-8 first because some servers
            # choose to localize their reason strings. If the string
            # isn't utf-8, we fall back to iso-8859-1 for all other
            # encodings. (See PR #3538)
            try:
                reason = self.reason.decode('utf-8')
            except UnicodeDecodeError:
                reason = self.reason.decode('iso-8859-1')
        else:
            reason = self.reason
    
        if 400 <= self.status_code < 500:
            http_error_msg = u'%s Client Error: %s for url: %s' % (self.status_code, reason, self.url)
    
        elif 500 <= self.status_code < 600:
            http_error_msg = u'%s Server Error: %s for url: %s' % (self.status_code, reason, self.url)
    
        if http_error_msg:
>           raise HTTPError(http_error_msg, response=self)
E           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://spreadsheets.google.com/feeds/list/1eEa2ra2yHBXVZ_ctH4J15tFSGEu-VTSunsrvaCAV598/od6/public/values?alt=json

.tox/python/lib/python3.6/site-packages/requests/models.py:953: HTTPError

```

So it must be an issue with spreadsheets occasionally returning not found... maybe a bug in the spreadsheet.

For now, I'll just hobble the test.

**Comment by jaraco** [comment]
That spreadsheet appears not to be available for me at all now. I'm now guessing that Google rolled out a change incrementally. I can see that the spreadsheet form of loading emergency compliments was added by me in b1c3b0e3fd3774b6153542aa734310829e858333.

**Comment by jaraco** [comment]
I tried to replicate the technique that was used and found that [even this published technique](https://www.freecodecamp.org/news/cjn-google-sheets-as-json-endpoint/) no longer works, so it does appear as if Google has turned down this feature.

**Comment by jaraco** [comment]
I don't even recall where that spreadsheet came from. It seems I don't currently have access to it.

**Comment by jaraco** [comment]
Ooh. It looks like others have had the same problem, with a [possible solution](https://stackoverflow.com/a/71334523/70170).

**Comment by jaraco** [comment]
It's annoying that this new technique requires a Google API key and requires that key to be configured with access to the Google Sheets API, so even though I already have a pmxbot testing and pmxbot API, they weren't authorized to use the API without a couple of steps. Now they're configured and I've confirmed that a technique does work, but it's probably more trouble than anyone will bother to use.

Also, I'd lost access to the compliments that were in the sheet, so I've created a new sheet. Internet is hard.
